code/index.html

```diff
@@ -1,30 +1,152 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="utf-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Project 3</title>
-    <!-- dont forget to add a css file and link it here! -->
-  </head>
-  <body>
-    <!-- video or image as a header is cool :) -->
-
-    <!-- Signup form -->
-
-    <section class="accordion">
-      <!-- Accordion -->
+
+<head>
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Go Green</title>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link
+    href="https://fonts.googleapis.com/css2?family=Source+Sans+Pro:ital,wght@0,200;0,300;0,400;0,600;0,700;0,900;1,200;1,300;1,400;1,600;1,700;1,900&display=swap"
+    rel="stylesheet">
+  <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+
+  <!-- Hero Header -->
+  <header id="hero" class="hero-wrapper-container hero-styling">
+    <div class="hero-inner-container hero-video">
+      <video class="video" id="video" loop muted autoplay playsinline>
+        <source src="forest.mp4" type="video/mp4">
+      </video>
+    </div>
+    <div class="title title-container">
+      <h1>Lets shape the future together!</h1>
+      <p>Become a volunteer today</p>
+      <button class="btn">Learn more</button>
+    </div>
+  </header>
+
+  <!-- Signup form -->
+  <div class="info-section-wrapper">
+    <form id="form" class="wrapper-container" action="http://httpbin.org/anything" method="POST" autocomplete="on">
+
+      <!-- first three inputs -->
+      <section class="inputs-wrapper">
+        <label class="child-form-container" for="name">
+          <input class="input custom-input" id="name" type="text" name="name" placeholder="Name" value=""
+            pattern="[a-zA-Z ]{1-32}" oninvalid="setCustomValidity('Please enter alphabets only.')" required>
```

Comment:
WoW! I was thinking I wanted to add something like this. Now I learned from you. Great work! 
Can I ask, the Value="" does not have any data, should it maybe be removed?